pkg/util/scrubber/scrubber.go

```diff
@@ -159,3 +168,38 @@ func (c *Scrubber) scrub(data []byte, replacers []Replacer) []byte {
 	}
 	return data
 }
+
+// gitleaks applies the gitleaks library to the given data.
+func (c *Scrubber) gitleaks(data []byte) []byte {
```

Comment:
non-blocking nit: `scrubber.gitleaks` doesn't really intuitively imply that we are scrubbing data. Because of that, I think maybe a bit more explicit naming of it would be better for maintainability (e.g. `gitleaksScrub` or something like that).